src/domain/submission/services/submission-service.ts

```diff
@@ -126,6 +126,27 @@ export class SubmissionService {
             });
         return this.get(id);
     }
+
+    async resubmit(submission: Submission): Promise<void> {
+        try {
+            await this.runMecaExport(submission, '0.0.0.0');
+            submission.status = SubmissionStatus.MECA_EXPORT_SUCCEEDED;
+        } catch (e) {
+            logger.error('Unable to resubmit', e);
```

Comment:
I assume you don't set the status here to failed as it's already in the failed state?

Reply:
Yup, seems to be the case.

Reply:
Correct 👍